crates/cli_utils/src/bench_utils.rs

```diff
@@ -107,7 +107,10 @@ fn bench_cmd<T: Measurement>(
 
 pub fn bench_nqueens<T: Measurement>(bench_group_opt: Option<&mut BenchmarkGroup<T>>) {
     exec_bench_w_input(
-        &file_path_from_root("crates/cli_testing_examples/benchmarks", "nQueens.roc"),
+        &file_path_from_root(
+            "crates/cli/tests/cli_testing_examples/benchmarks",
```

Comment:
Looks like the paths in this file still need to be updated

Reply:
Good catch, updated.